Headlines/src/Controllers/OrientationAwareControllers.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  OrientationAwareControllers.swift
+//  Headlines
+//
+//  Created by Julio Carrettoni on 5/4/17.
+//  Copyright © 2017 Ezequiel Becerra. All rights reserved.
+//
+
+import UIKit
+
+class OrientationAwareTabBarController: UITabBarController {
+    override open var preferredInterfaceOrientationForPresentation: UIInterfaceOrientation {
+        return .portrait
+    }
+    
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            return .all
+        }
+        else {
+            return .portrait
+        }
+    }
+    
+    override var shouldAutorotate: Bool {
+        get { return UIDevice.current.userInterfaceIdiom == .pad }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f609a77816dcd1d12fd0f37d58964ee47f86c075/Headlines/src/Controllers/OrientationAwareControllers.swift#L26"></span>Computed read-only properties should avoid using the get keyword.</td>
    </tr>
  </tbody>
</table>